CMakeLists.txt

```diff
@@ -200,8 +200,22 @@ if(WITH_LIBUNWINDSTACK)
 	endif()
 endif()
 
-if(WIN32)
+if(MSVC)
 	target_link_libraries(sentry PRIVATE dbghelp pathcch)
+elseif(MINGW)
+	
+	target_compile_definitions(sentry PRIVATE 
+		__LP64__ #force 64bits flag to remove some -Wincompatible-pointer-types errors 
```

Comment:
hm, I don’t think this is such a good idea, doesn’t this define have a lot more implications?
I made a pass through all the warnings recently and fixes quite a few.

Reply:
You are right, the latest MinGW headers state now clearly that __LP64__ should be absent on 64bits builds... I need to find a workaround.

Reply:
Well if ignoring the warnings is ok for now, I updated the compiler options accordingly.